.travis/smoke-test.sh

```diff
@@ -10,6 +10,7 @@ echo "Wait for containers health"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_blog-articles_web_1"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_scholarly-articles_fpm_1"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_scholarly-articles_web_1"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_search_elasticsearch_1"
```

Comment:
Can this script be updated to loop through a list of service names?

Reply:
Should be easy